lib/tokenizers/decoder/bpe.ex

```diff
@@ -0,0 +1,7 @@
+defmodule Tokenizers.Decoder.BPE do
+  @doc """
+  Creates new BPE decoder
+  """
+  @spec new(suffix :: String.t()) :: {:ok, Tokenizers.Decoder.t()} | {:error, any()}
+  defdelegate new(suffix \\ "</w>"), to: Tokenizers.Native, as: :decoders_bpe
```

Comment:
I am not very happy with one module per decoder. Could we instead have `Tokenizers.Decoder.bpe`, `Tokenizers.Decoder.byte_fallback`, etc? The smaller abstraction in Elixir is a function and one function modules are frequently an anti-pattern.

Reply:
And yes, I agree Models should be modules :)